src/libraries/System.IO.Compression.ZipFile/src/Resources/Strings.resx

```diff
@@ -1,4 +1,64 @@
+<root>
+  <!-- 
+    Microsoft ResX Schema 
+    
+    Version 2.0
+    
+    The primary goals of this format is to allow a simple XML format 
+    that is mostly human readable. The generation and parsing of the 
+    various data types are done through the TypeConverter classes 
+    associated with the data types.
+    
+    Example:
+    
+    ... ado.net/XML headers & schema ...
+    <resheader name="resmimetype">text/microsoft-resx</resheader>
+    <resheader name="version">2.0</resheader>
+    <resheader name="reader">System.Resources.ResXResourceReader, System.Windows.Forms, ...</resheader>
+    <resheader name="writer">System.Resources.ResXResourceWriter, System.Windows.Forms, ...</resheader>
+    <data name="Name1"><value>this is my long string</value><comment>this is a comment</comment></data>
+    <data name="Color1" type="System.Drawing.Color, System.Drawing">Blue</data>
+    <data name="Bitmap1" mimetype="application/x-microsoft.net.object.binary.base64">
+        <value>[base64 mime encoded serialized .NET Framework object]</value>
+    </data>
+    <data name="Icon1" type="System.Drawing.Icon, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
+        <value>[base64 mime encoded string representing a byte array form of the .NET Framework object]</value>
+        <comment>This is a comment</comment>
+    </data>
+                
+    There are any number of "resheader" rows that contain simple 
+    name/value pairs.
+    
+    Each data row contains a name, and value. The row also contains a 
+    type or mimetype. Type corresponds to a .NET class that support 
+    text/value conversion through the TypeConverter architecture. 
+    Classes that don't support this are serialized and stored with the 
+    mimetype set.
+    
+    The mimetype is used for serialized objects, and tells the 
+    ResXResourceReader how to depersist the object. This is currently not 
+    extensible. For a given mimetype the value must be set accordingly:
+    
+    Note - application/x-microsoft.net.object.binary.base64 is the format 
+    that the ResXResourceWriter will generate, however the reader can 
+    read any of the formats listed below.
+    
+    mimetype: application/x-microsoft.net.object.binary.base64
+    value   : The object must be serialized with 
+            : System.Runtime.Serialization.Formatters.Binary.BinaryFormatter
+            : and then encoded with base64 encoding.
+    
+    mimetype: application/x-microsoft.net.object.soap.base64
+    value   : The object must be serialized with 
+            : System.Runtime.Serialization.Formatters.Soap.SoapFormatter
+            : and then encoded with base64 encoding.
+
+    mimetype: application/x-microsoft.net.object.bytearray.base64
+    value   : The object must be serialized into a byte array 
+            : using a System.ComponentModel.TypeConverter
+            : and then encoded with base64 encoding.
+    -->
```

Comment:
```suggestion
<root>
```

Reply:
I see a lot of resx files in our repo that preserve the automatically-added info:

- https://github.com/dotnet/runtime/blob/main/src/libraries/System.Memory/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.Console/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.IO.Pipes/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Http/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.Formats.Tar/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.Xml/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.IO.Packaging/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.Speech/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.IO.Ports/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.CSharp/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.Text.Json/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.IO.Hashing/src/Resources/Strings.resx

I see fewer examples of resx files that do not include that info:

- https://github.com/dotnet/runtime/blob/main/src/libraries/System.Linq/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.CodeDom/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Mail/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.Threading/src/Resources/Strings.resx
- https://github.com/dotnet/runtime/blob/main/src/libraries/System.ObjectModel/src/Resources/Strings.resx

Reply:
Right, we are inconsistent here but it shows up as added in your diff. For this PR, please undo the change.

Reply:
Ok, I can remove it. Do you know who the owner of VS resx is?

Reply:
Unfortunately not. @ericstj @danmoseley would you know?

Reply:
@drewnoakes owns the designer: https://github.com/dotnet/project-system/tree/main/src/Microsoft.VisualStudio.Editors/ResourceEditor

It uses the writer which is owned by @dotnet/dotnet-winforms.

In .NETFramework it writes the larger comment: 
https://referencesource.microsoft.com/#System.Windows.Forms/winforms/Managed/System/Resources/ResXResourceWriter.cs,125

In .NET (Core) it was reduced:
https://github.com/dotnet/winforms/blob/c2bfdb79d6b584a53b6811040f7103413426cff6/src/System.Windows.Forms/src/System/Resources/ResXResourceWriter.cs#L44

Though the designer still runs on NetFx.

Reply:
I say -- let's do this :)

Reply:
If an option is created to control this, we (dotnet/project-system) can make sure it's exposed in the Properties pane in VS for SDK-style projects, likely setting metadata on the MSBuild item. The CSPROJ team would need to do something similar for legacy projects. It's possible that other project flavours would need to be changed to support this everywhere.

I've long wanted to improve the generated comment in the `.Designer.cs` file too, to make it clearer what the string actually is (such as with a `<c>` tag) but again was concerned about pushback from zillions of diffs for something that's really just a cosmetic quality-of-life thing, and doesn't impact running code. For the `.Designer.cs` file, I assume doing this in a source generator would be a better option, but that doesn't help with the `.resx` comment/XSD.

Reply:
> If an option is created to control this

Do we need an option? The idea in my mind is that it's removed from all templates and when editing via ResXResourceWriter or the designer (I don't recall whether that uses ResxResourceXXX or its own code) it is only written out if it previously existed. Would that work?

I agree that the Designer.cs should become a source generator eventually, but that is orthogonal other than touching many of the same components.

Reply:
@drewnoakes I would like to open an issue in the right repo to continue the conversation, I'll copy what was mentioned here. I assume dotnet/project-system is the right place?